src/modules/products/ui/components/tags-filter.tsx

```
@@ -0,0 +1,72 @@
+      {isLoading ? (
+        <div className="flex items-center justify-center p-4">
+          <LoaderIcon className="size-4 animate-spin" />
+        </div>
+      ) : (
+        data?.pages.map((page) =>
+          page.docs.map((tag) => (
+            <div
+              key={tag.id}
+              className="flex items-center justify-between cursor-pointer"
+              onClick={() => onClick(tag.name)}
+            >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)